november-2017/php-version.md

```diff
@@ -0,0 +1,6 @@
+### Upcoming MyBB Releases And Their Minimum Supported PHP Versions
+Frequent visitors to the MyBB Community Forums may remember [a discussion from July 2016](https://community.mybb.com/thread-195675.html) regarding the minimum PHP version required to use MyBB 2.0. The benefits of such a change were detailed in great length by [Devilshakerz](https://community.mybb.com/user-47371.html).
```

Comment:
```diff
-Frequent visitors to the MyBB Community Forums may remember [a discussion from July 2016](https://community.mybb.com/thread-195675.html) regarding the minimum PHP version required to use MyBB 2.0. The benefits of such a change were detailed in great length by [Devilshakerz](https://community.mybb.com/user-47371.html).
+Frequent visitors to the MyBB Community Forums may remember [a discussion from July 2016](https://community.mybb.com/thread-195675.html) regarding the minimum PHP version that MyBB 2.0 would require. The intentions were that MyBB 2.0 would require PHP 7.0 and other environment requirements could be changed with each minor release (for example from 2.3 to 2.4).
```

Clean up current details.


First paragraph should be expanded a little more to introduce what the blog post is about it is currently rather disjointed from the rest of the post. I suggest including the information about the 1.10 announcement here then discuss the exact details in the second paragraph as at present.

Reply:
That first paragraph is my fault with the changes I amde last night. I couldn't really think of anything to add.